assets/src/edit-story/components/inspector/documentInspector.js

```diff
@@ -1,9 +1,187 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useEffect } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app/story';
+import { useConfig } from '../../app/config';
+import UploadButton from '../uploadButton';
+import useInspector from './useInspector';
+import { SelectMenu, InputGroup } from './shared';
+
+const ButtonCSS = css`
+	color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	font-size: 14px;
+	width: 100%;
+	padding: 15px;
+	background: none;
+	margin: 5px 0px;
+`;
+const Img = styled.img`
+	width: 100%;
+	max-height: 300px
+`;
+
+const Group = styled.div`
+	border-color: ${ ( { theme } ) => theme.colors.mg.v1 };
+	display: block;
+	align-items: center;
+	margin: 15px 0px;
+`;
+
+const RemoveButton = styled.button`
+	${ ButtonCSS }
+`;
 
 function DocumentInspector() {
```

Comment:
This component is way too big and should be split up somehow. I don't know what would be a natural split off the top of my head, but it should have concerns split in several smaller components.

Reply:
I think you are correct here. There is a lot and is makes it a little hard to follow. But until we know the final UX, it seems unwise to split this now. This PR is just so we can get the functionality in, we can style it up and move it around later.